pkg/operator/controller/prometheus.go

```diff
@@ -46,6 +46,8 @@ const (
 	monitorName         = "service-monitor-cdi"
 	defaultMonitoringNs = "monitoring"
 	runbookURLBasePath  = "https://kubevirt.io/monitoring/runbooks/"
+	partOfLabelValue    = "kubevirt"
```

Comment:
I was thinking if we could have `Alert\Metric` as part of the variable name to indicate these are not the standard relationship label values

Reply:
@assafad Please update the names 
partOfAlertLabelValue and componentAlertLabelValue